src/mlagility/cli/cloud/requirements.txt

```diff
@@ -0,0 +1,4 @@
+azure-mgmt-resource
+azure-mgmt-compute
+azure-mgmt-network
+azure-identity
```

Comment:
Specify versions.

Reply:
Added `>=` requirements in https://github.com/groq/mlagility/pull/298/commits/f8cb91518d5c2a3a9907ec74687d48d55d63022a